Tests/UnitTests/Plugins/GeoModel/GeoTubeToVolumeTest.cpp

```diff
@@ -0,0 +1,62 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+// clang-formal off
+#include "Acts/Plugins/GeoModel/GeoModelDetectorObjectFactory.hpp"
+// clang-formal on
```

Comment:
Add comment here why we switch off clang format (in order to avoid conflicts with declarations in Geomodel that is fixed in v3.5 -->https://gitlab.cern.ch/GeoModelDev/GeoModel/-/merge_requests/351)